src/reference/asciidoc/aggregator.adoc

```diff
@@ -249,11 +232,13 @@ Here is an example of how to configure an aggregator with the previous implement
 
 <bean id="releaser" class="SimpleSequenceSizeReleaseStrategy" />
 ----
+====
 
 ===== CorrelationStrategy
```

Comment:
Must be wrapped into code as you do in other headers when it is about a class name.